inc/sainat.h

```diff
@@ -114,6 +96,20 @@ typedef enum _sai_nat_entry_attr_t
      */
     SAI_NAT_ENTRY_ATTR_SRC_IP_MASK,
 
+    /**
+     * @brief Replace destination IPv4 address in packet.
```

Comment:
Replace virtual router associated with the IPv4 addresses in the packet.

Reply:
done